front/src/pages/RegistrationPage.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect, useContext } from "react";
+import { useNavigate, Link } from "react-router-dom";
+import { AppContext } from "../context/AppContext";
+import { validationService } from "../services/validationService";
+import { messageService } from "../services/messageService";
+import Loader from "../components/Loader";
+import { authService } from "../services/authService.js";
+import EmailField from "../components/EmailField";
+import PasswordField from "../components/PasswordField";
+import "../styles.scss";
+
+const RegistrationPage = ({ addClasses = "" }) => {
+  //#region Get setUser function from context
+  const { user } = useContext(AppContext);
+  //#endregion
+
+  //#region Get navigation from router
+  const navigate = useNavigate();
+  //#endregion
+
+  //#region UserData
+  const [userData, setUserData] = useState(null);
+  //#endregion
+
+  //#region Handling form fields editing
+  const changeInputValueHandler = (fieldName, newValue) => {
```

Comment:
Consider getting a fieldName attribute from the event target name

Reply:
Fixed